cgotorch/cgotorch.h

```diff
@@ -45,13 +46,9 @@ void Dataset_Stack(Dataset dataset, Transform transform);
 typedef void *Iterator;
 typedef void *DataLoader;
 
-typedef struct Data {
-  Tensor Data;
-  Tensor Target;
-} Data;
-
+void Loader_Close(DataLoader loader);
 Iterator Loader_Begin(DataLoader loader);
-Data Loader_Data(Iterator iter);
+void Loader_Data(Iterator iter, Tensor data[]);
 bool Loader_Next(DataLoader loader, Iterator iter);
 DataLoader MakeDataLoader(Dataset dataset, int batchsize);
```

Comment:
Need a comment on the batchSize parameter. In the example, it is set to 0 -- what does 0 mean as a batch size?

Reply:
I see. That is because I removed clang-format from the pre-commit hooks.